### PR TITLE
DDF-4548 Update docs for enforced validator plugin

### PR DESCRIPTION
**Pull request by austinsteffes**
##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
Link to 2.13.x PR: #4564 

#### Who is reviewing it? 
@tyler30clemens 
@jhunzik 

#### Select relevant component teams: 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@brjeter
@ricklarsen - Documentation
@vinamartin

#### How should this be tested?
Build successful
#### What are the relevant tickets?

For GH Issues:
Fixes: #4548 

#### Screenshots

Screen Shot 2019-04-15 at 11 18 04 AM

#### Checklist:
- [X ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by tyler30clemens** [comment]
I notice this has a different parent doc and ordering from #4564. Is this on purpose / in the correct place?

**Comment by ricklarsen** [comment]
> I notice this has a different parent doc and ordering from #4564. Is this on purpose / in the correct place?

I haven't checked this out to build/confirm yet, but I previously moved a large chunk of this content on this branch, so it has diverged from the other branch quite a bit. 

tl;dr: probably

**Comment by austinsteffes** [comment]
@tyler30clemens fixed to what you had said above

**Comment by austinsteffes** [comment]
build now

File: distribution/docs/src/main/resources/content/_managing/_running/forms-commands.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 :status: published
 :summary: External methods of creating and managing system search forms.
 :parent: Ingesting Data
-:order: 03
+:order: 04
```
Comment (tyler30clemens) [inline distribution/docs/src/main/resources/content/_managing/_running/forms-commands.adoc]: Is the order for `Ingesting Data` still changed?

File: distribution/docs/src/main/resources/content/_managing/_running/other-ingest.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 :status: published
 :summary: Other methods of ingesting data.
 :parent: Ingesting Data
-:order: 04
+:order: 05
```
Comment (tyler30clemens) [inline distribution/docs/src/main/resources/content/_managing/_running/other-ingest.adoc]: Is the order for `Ingesting Data` still changed?

Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/other-ingest.adoc]: yes, there were two that were `03` previously

File: distribution/docs/src/main/resources/content/_managing/_running/forms-commands.adoc (hunk 1 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/forms-commands.adoc]: yes, there were two that were `03` previously

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
+depending on whether *Enforce errors* is selected within the admin console. Below is a list
+of the validators run against the data ingested.
+
+[NOTE]
+====
+To navigate to turn on/off *Enforce errors*:
+ . Navigate to the *${admin-console}*.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: Is there an extra space at the start of this line? it could affect your list format.

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+. Select the *System* tab.
+. Select the *Configuration* tab.
+. Select *Metacard Validation Marker Plugin*.
+.. If *Enforce errors* is checked, these validators below will be run on ingest.
+.. If *Enforce errors* is not checked, these validators below will *not* be run on ingest.
+====
+
+=== List of Validators run on ingest
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: Can you make this title case? and I don't think we need the "List of" at the beginning

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
+:title: Validator Plugins on Ingest
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 00
+:summary: Validator plugins run on Ingest
+
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: I think we can simplify this sentence. Maybe "When *Enforce Errors* is enabled within the ${admin-console}, validator plugins ensure the data being ingested is valid."

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+depending on whether *Enforce errors* is selected within the admin console. Below is a list
+of the validators run against the data ingested.
+
+[NOTE]
+====
+To navigate to turn on/off *Enforce errors*:
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: We could make this a `.Title` above the `[NOTE]` line. I also think we can cut the `To navigate to` and just make it `Enforcing Errors`

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+. Select the *System* tab.
+. Select the *Configuration* tab.
+. Select *Metacard Validation Marker Plugin*.
+.. If *Enforce errors* is checked, these validators below will be run on ingest.
+.. If *Enforce errors* is not checked, these validators below will *not* be run on ingest.
+====
+
+=== List of Validators run on ingest
+
+* TDF Schema Validation Service
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: ✏️ We could simplify the syntax of this list by doing 
`TDF Schema Validation Service:: This service validates a TDO against a TDF schema.`

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc (hunk 5 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc (hunk 4 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc (hunk 3 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc (hunk 6 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc (hunk 7 above)
Comment (austinsteffes) [reply distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc]: fixed

**Comment by austinsteffes** [comment]
build now

**Comment by austinsteffes** [comment]
build now

**Comment by brjeter** [comment]
Manually verified the rendered HTML document ✅
